INSTALL_LINUX.md

```diff
@@ -129,10 +130,26 @@ git submodule update -i --recursive
 ### Download OpenColorIO-Configs
 
 In the past, OCIO configs were a submodule, though due to the size of the repository, we have chosen instead
-to make a tarball release and let you download it [here](https://github.com/NatronGitHub/OpenColorIO-Configs/archive/Natron-v2.1.tar.gz).
-Place it at the root of Natron repository.
+to make a tarball release and let you download it [here](https://github.com/NatronGitHub/OpenColorIO-Configs/archive/Natron-v3.0.tar.gz).
```

Comment:
The tag Natron-v3.0 is for the master branch (but is the same as Natron-v2.3 anyway)
The correct version is Natron-v2.3

Reply:
ok I will fix this. I saw that the v3 where the same than on the portable version of natron so I assume this was the correct one. didn't saw the v2.3 was the same